Gems/Multiplayer/Code/Source/Components/NetworkCharacterComponent.cpp

```diff
@@ -113,17 +113,24 @@ namespace Multiplayer
         // During activation the character controller is not created yet.
         // Connect to CharacterNotificationBus to listen when it's activated after creation.
         Physics::CharacterNotificationBus::Handler::BusConnect(GetEntityId());
+
+        // Set up the network event handlers. These can be bound before the character controller is created
+        // because they check for the character controller to exist inside of the handlers.
+
```

Comment:
remove newline between the comment and code?

Reply:
It's a style choice. I tend to use a newline separator if the comment refers to a series of lines that follow it, and no newline if it only refers to a single line.